ruby_04-apis_and_scalability/js_refactor_tractor.md

```diff
@@ -303,6 +303,141 @@ __Discussion Points__
 
 #### Dead Code Among the Living
 
+#### If If If Else If Else Else If Else ...
+
+Deeply nested if/else statements should be avoided-- they make a program difficult to follow. There are many ways to refactor/restructure to avoid this condition, and good strategy can vary with the situation.
+
+##### 1. Break up into functions.
```

Comment:
Looks like #### (4) for topic headers, and #####(5) for the three if/else subtopics? Let me know if it doesn't look right.